src/eth_provider/provider.rs

```diff
@@ -574,18 +574,17 @@ where
         #[cfg(feature = "hive")]
         self.deploy_evm_transaction_signer(signer).await?;
 
-        // Convert the transaction to a Starknet transaction
-        let transaction = to_starknet_transaction(&transaction_signed, maybe_chain_id, signer, max_fee)?;
-
-        // Add the transaction to the Starknet provider
-        let res = self.starknet_provider.add_invoke_transaction(transaction).await.map_err(KakarotError::from)?;
-
         // Serialize transaction document
         let transaction =
             from_recovered(TransactionSignedEcRecovered::from_signed_transaction(transaction_signed.clone(), signer));
 
         // Update or insert the pending transaction in the database
         if let Some(pending_transaction) = pending_transaction {
+            tracing::info!(
+                "Transaction with hash {:?} already exists in the pending pool, updating the retries count to {}.",
```

Comment:
Could we maybe implement display for the pending transaction type?

Reply:
Yes agreed, fixed

Reply:
ah sorry I should have explained more: I think we can implement `Display` trait for the `StoredPendingTransaction` struct and then reduce this to `tracing::info!("updating {}. Retries: {}")`. I think we should be careful not to log too much, since this is all stored on Karnot's side.

Reply:
Just reduced the text as discussed